src/compile/model.ts

```diff
@@ -311,12 +310,10 @@ export abstract class Model {
     const {style: _, ...baseView} = view;
 
     const e = {};
-    for (const property in baseView) {
-      if (hasOwnProperty(baseView, property)) {
-        const value = baseView[property];
-        if (value !== undefined) {
-          e[property] = {value};
-        }
+    for (const property of keys(baseView)) {
+      const value = baseView[property];
+      if (value !== undefined) {
+        e[property] = isSignalRef(value) ? value : {value};
```

Comment:
there is a helper method that we use everywhere.  valueOrSignal or the other way round.

Reply:
`signalOrValueRef`, which seems like the wrong name. I'll send a pull request.